src/components/views/messages/MPollBody.tsx

```diff
@@ -18,48 +18,174 @@ import React from 'react';
 import { _t } from '../../../languageHandler';
 import { replaceableComponent } from "../../../utils/replaceableComponent";
 import { IBodyProps } from "./IBodyProps";
-import { IPollAnswer, IPollContent, POLL_START_EVENT_TYPE } from '../../../polls/consts';
+import {
+    IPollAnswer,
+    IPollContent,
+    IPollResponse,
+    POLL_RESPONSE_EVENT_TYPE,
+    POLL_START_EVENT_TYPE,
+} from '../../../polls/consts';
 import StyledRadioButton from '../elements/StyledRadioButton';
+import { MatrixEvent } from "matrix-js-sdk/src/models/event";
+import { Relations } from 'matrix-js-sdk/src/models/relations';
+import { MatrixClientPeg } from '../../../MatrixClientPeg';
 
 // TODO: [andyb] Use extensible events library when ready
 const TEXT_NODE_TYPE = "org.matrix.msc1767.text";
 
 interface IState {
     selected?: string;
+    pollRelations: Relations;
 }
 
 @replaceableComponent("views.messages.MPollBody")
 export default class MPollBody extends React.Component<IBodyProps, IState> {
     constructor(props: IBodyProps) {
         super(props);
 
-        this.state = {
-            selected: null,
-        };
+        const pollRelations = this.fetchPollRelations();
+        let selected = null;
+
+        const userVotes = collectUserVotes(allVotes(pollRelations), null);
+        const userId = MatrixClientPeg.get().getUserId();
+        const currentVote = userVotes.get(userId);
+        if (currentVote) {
+            selected = currentVote.answers[0];
+        }
+
+        this.state = { selected, pollRelations };
+
+        this.addListeners(this.state.pollRelations);
+        this.props.mxEvent.on("Event.relationsCreated", this.onPollRelationsCreated);
+    }
+
+    componentWillUnmount() {
+        this.props.mxEvent.off("Event.relationsCreated", this.onPollRelationsCreated);
+        this.removeListeners(this.state.pollRelations);
+    }
+
+    addListeners(pollRelations?: Relations) {
```

Comment:
explicit member access please (public/private)

Reply:
I added a commit for this, thanks.